docs/USER-DOCUMENTATION.md

```diff
@@ -149,6 +149,14 @@ In Windows:
 set ETCHER_DISABLE_UPDATES=1
 ```
 
+Simulate an update alert
+------------------------
+
+You can set the `ETCHER_LATEST_VERSION` environment variable to a valid semver
+version to trick the application that what you put there is the latest
+available version, therefore causing the update notification dialog to be
+presented at startup.
```

Comment:
"presented at startup (if the `ETCHER_LATEST_VERSION` environment variable is greater than the Etcher version)."

?
Bit verbose, but more accurate ;)